build/web_compose/Dockerfile

```diff
@@ -66,6 +66,8 @@ RUN pip3 install -r /workspace/server/requirements.txt
 COPY nl_app.py /workspace/nl_app.py
 COPY deploy/nl/. /datacommons/nl/
 COPY nl_server/requirements.txt /workspace/nl_server/requirements.txt
+# TODO: Undo this once we figure out why lancedb breaks Custom DC Docker
+RUN sed -i '' '/lancedb/d' /workspace/nl_server/requirements.txt
```

Comment:
Can we skip removing this since it works with requirements.txt as-is?

Reply:
I don't understand why that is?

Reply:
Also, did you delete any lancedb.py file and refs, vs. keep them but not have the code run?

Reply:
I ran with lancedb in requirements.txt but without refs and without lancedb.py